tests/dataframe_tests/test_dataframe_aggregation.py

```diff
@@ -1,52 +1,62 @@
+import pytest
 import tube.settings as config
 from tests.dataframe_tests.util import (
     get_spark_session,
     initialize_mappings,
     get_translator,
     assert_dataframe_equality,
-    get_input_output_dataframes
+    get_input_output_dataframes,
+    mock_dictionary_url
 )
 from tube.utils.general import get_node_id_name
 
-DICTIONARY_URL = "https://s3.amazonaws.com/dictionary-artifacts/ibdgc-dictionary/1.6.10/schema.json"
-initialize_mappings()
+initialize_mappings("ibdgc")
 
-def test_get_direct_children_with_parent(spark_context):
+def test_get_direct_children_with_parent(spark_context, schema_context):
     input_df, expected_df = get_input_output_dataframes(
         get_spark_session(spark_context),
+        "ibdgc",
         "participant__0_Translator.translate_parent",
         "participant__0_Translator.get_direct_children"
     )
-    translator = get_translator(spark_context, config, "participant", "aggregation")
+    with schema_context as _schema:
+        _schema.return_value = mock_dictionary_url("ibdgc")
+        translator = get_translator(spark_context, config, "ibdgc", "participant", "aggregation")
```

Comment:
I want to have the parameter so that we can make change on the value of parameter of `mock_dictionary_url` function depending on the test @paulineribeyre

Reply:
maybe it could still be a reusable fixture or function, "set_dictionary_to_ibdgc"

Reply:
actually it looks fine if you don't want to change it, i didn't see that `translator = get_translator(...)` was already repeated before this